### PR TITLE
Modify reboot count query

### PR DESCRIPTION
**Pull request by binoue**
I saw [this webpage](https://tips.weseek.co.jp/Tips/Prometheus/Kubernetes%28GKE%29%20%E3%81%A7%E4%BE%BF%E5%88%A9%E3%81%AA%20Prometheus%20%E5%B0%8F%E3%83%8D%E3%82%BF%E9%9B%86#Node%20%E3%81%AE%E5%86%8D%E8%B5%B7%E5%8B%95%E3%82%92%E6%A4%9C%E7%9F%A5%E3%81%99%E3%82%8B) and thought that using [this changes() function](https://prometheus.io/docs/prometheus/latest/querying/functions/) would show the reboot count more properly.
Therefore, this PR modified the current query to show each nodes' reboot timing by using `changes()` function.

Signed-off-by: binoue <banji-inoue@cybozu.co.jp>

### CODE REVIEW

File: monitoring/base/grafana-operator/dashboards/all-nodes.yaml [hunk 1]
```diff
@@ -719,7 +719,7 @@ spec:
                 "steppedLine": false,
                 "targets": [
                     {
-                        "expr": "deriv(time() - node_boot_time_seconds{}) < bool 0.1",
+                        "expr": "changes(node_boot_time_seconds{}[15m:]) > 0",
```
Comment (morimoto-cybozu) [inline monitoring/base/grafana-operator/dashboards/all-nodes.yaml:722]: I didn't know the notation of `[15m:]`.
Are you familiar with this type of query? Or did you constructed with the help of autocomplete?

```suggestion
                        "expr": "changes(node_boot_time_seconds[15m]) > 0",
```

Comment (ymmt2005) [reply monitoring/base/grafana-operator/dashboards/all-nodes.yaml]: @binoue 
> <resolution> is optional, which is equivalent to step in /query_range API.

It seems optional, isn't it?

Comment (ymmt2005) [reply monitoring/base/grafana-operator/dashboards/all-nodes.yaml]: Ah, you mean ':' may not be optional.

Comment (binoue) [reply monitoring/base/grafana-operator/dashboards/all-nodes.yaml]: @morimoto-cybozu 

Sorry. I was misunderstanding.
The resolution rate was for aggregating data with the lower resolution/range.

See [this official doc](https://prometheus.io/blog/2019/01/28/subquery-support/#motivation).
In this case, the range vector is not the aggregated data, so no need to specify the resolution.(i.e. no colon)

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Review by zoetrope — APPROVED** [review]

**Pull request merged**